lib/utils/socketChainHelper.js

```diff
@@ -70,7 +70,7 @@ const processServerResponse = toString => function processServerResponseHandler(
 		return false;
 	}
 	// eval fail
-	if (!isSuccess && isEval) {
+	if (!isSuccess && (isEval || (isTestLine && !isAssertionError))) {
```

Comment:
Personally I think it should either be AssertionError or unknown error or something like InvalidInput error. :), but not evaluation error because its not eval chain.

Reply:
I'm OK with invalidInput as well, as it seems to be the code we get from server. Please, discuss this with Lukas Lohnicky and then let's close the issue ;)